distribution/changelog.txt

```diff
@@ -27,6 +27,7 @@
 - Fix: [#15255] Tile Inspector shows banner information on walls that do not contain one.
 - Fix: [#15257] Chat icon shows in scenario/track editor. Other icons don't disable when deactivated in options menu.
 - Fix: [#15289] Unexpected behavior with duplicated banners which also caused desyncs in multiplayer.
+- Fix: [#15476] Crash in map_place_clear_func().
```

Comment:
I think this change log entry might be a bit too technical, maybe something like ```Potential crash placing/clearing scenery```?

Reply:
```suggestion
- Fix: [#15476] Crash when placing/clearing small scenery.
```